### PR TITLE
Restructure theming files for dc-repository theme

### PR DESCRIPTION
**Pull request by AKFosterDev**
This moves custom theming partials into the dc-repository folder and reverts default hyrax partials and updates the theme wireframe.

### CODE REVIEW

File: app/views/themes/dc_repository/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<footer class="navbar navbar-inverse site-footer">
```
Comment (ShanaLMoore) [inline app/views/themes/dc_repository/_footer.html.erb:1]: @AKFosterDev Should the header also be in this directory?

File: app/views/_logo.html.erb [hunk 2]
```diff
@@ -1,15 +1,10 @@
-  <%# OVERRIDE: Hyrax v3.4.1 Change logo image for small screens and add image tag for small logo %>
-  
-  <% if logo_image %>
-    <a id="logo" class="navbar-brand d-flex" href="<%= hyrax.root_path %>" data-no-turbolink="true">
+ <% if logo_image %>
```
Comment (kirkkwang) [inline app/views/_logo.html.erb]: very minor, but the spacing is looking off on this line, in fact it seems like the entire file is indented over

File: app/views/themes/dc_repository/_footer.html.erb (hunk 1 above)
Comment (AKFosterDev) [reply app/views/themes/dc_repository/_footer.html.erb:1]: The top header is comprised of _masthead.html.erb, _logo.html.erb and _user_util_links.html.erb. Here is the [PR](https://github.com/scientist-softserv/utk-hyku/commit/47a44044a263123bd8bbab4398c0c3454e197b1d#diff-b2d655c9df0b3f1ba15f24c5a34aea9ee6a293be16b71361c3f0b169ac4756f2) for the changes made for creating the header.

Did you see a specific file?

File: app/views/_logo.html.erb (hunk 2 above)
Comment (AKFosterDev) [reply app/views/_logo.html.erb]: Good catch. Thanks. I just pushed the correction. Commit [d9d9587](https://github.com/scientist-softserv/utk-hyku/pull/210/commits/d9d9587474ea133e149af61a0259d3c883ba5e90).

File: app/views/themes/dc_repository/_footer.html.erb (hunk 1 above)
Comment (ShanaLMoore) [reply app/views/themes/dc_repository/_footer.html.erb:1]: ah gotcha. thanks for clarifying.

**Comment by ShanaLMoore** [comment]
@AKFosterDev So with these changes, if a user switched back to the default theme they won't see the customizations done (ie header)?

**Comment by AKFosterDev** [comment]
@ShanaLMoore

Correct. The default homepage looks like this:
Screenshot 2022-11-21 at 9 32 31 AM

**Review by ShanaLMoore — APPROVED** [review]

**Pull request merged**